### PR TITLE
orange & red color adjustments

### PR DESCRIPTION
**Pull request by michac789**
- Replace Orange S42 with S21, change corresponding components that used S42 to S21
- Change the color of red B99 and B100 
(question: is the HSB value in Figma important? now I just did base on the hex code, let me know if it is needed anywhere)
~~(question: not sure about the first task: 'Align the color of info icon in Alert with the one in Banner', asked this in Jira)~~
- Change color of alert info icon, make it the same as banner

Preview here: WAITING...

Edit:
The automated Github Action test failed, it comes from IndexTable which I didn't change.

Edit2:
**DO NOT REVIEW FIRST**
There are some strange issues when I updated the snapshot. Some snapshot changed into a loading screen which is weird, and some others have a square icon appearing on the bottom left of the screen.

### CODE REVIEW

**Comment by michac789** [comment]
currently checking the failed test case...

File: src/@next/utilities/colors/red.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const Brand = '#EC272B';
-export const B100 = '#FFC7C2';
-export const B99 = '#FC6568';
+export const B100 = '#FFEEEE';
```
Comment (chiahou) [inline src/@next/utilities/colors/red.ts]: @michac789 I just realized that the red-B100 appears too subtle in the badge component. Would it be possible to adjust red-B100 from #FFEEEE to #FFE5E6?

Comment (michac789) [reply src/@next/utilities/colors/red.ts]: sure!

**Comment by michac789** [comment]
This is very strange. 
Here is a picture for reference. The left one was the old color and the right one here is supposed to be the new color:
(for avatar)
image

In the preview, it works fine (showing the right color):
image
However, during the visual test image, it does not update anything:
image
(still showing the old color, the image does not get updated even with the `--update-snapshots` flag)
And oddly enough, it passed the test both locally and the CI in GitHub.
Additionally, there are updates in other parts instead such as `actionList`, which should not change.

**Review by chiahou — COMMENTED** [review]
@michac789 @ninariccimarie 
Overall, it looks good to me, but I did notice that the sequence of the red and orange colors got mixed up on the storybook. May need some fixing here.
colors

**Comment by michac789** [comment]
> @michac789 @ninariccimarie Overall, it looks good to me, but I did notice that the sequence of the red and orange colors got mixed up on the storybook. May need some fixing here. colors

checking on it now...

Edit:
May I clarify @chiahou , what you mean by the sequence? Is it that you expect the orange colors (left to right) to be ordered from S21, S64, ..., until S100 (in increasing numbers)? Or is it by increasing shades or based on the Figma coloring [here](https://www.figma.com/file/87fhOdF8vk1Uiy20bCqS85/Sirius-Web---UI-Kit-3.0-%5BQ4-2022%5D?type=design&node-id=4633-49902&t=tbDAnhaVyB5lqNCe-0)? Thank you.

I believe that the current ordering is based on alphabetical order (in this case, S100 comes first compared to S21, unless S21 is changed to S100), and is the same as the one deployed (the only difference is only in the color that is changed). But let me try to see if we can change the ordering sequence.

Edit 2:
Let me know @chiahou what sequence you prefer there, is it based on the Figma or any other sequence. I can edit those colors ordering, for example (red and orange based on increasing number):
image

**Comment by chiahou** [comment]
> > @michac789 @ninariccimarie Overall, it looks good to me, but I did notice that the sequence of the red and orange colors got mixed up on the storybook. May need some fixing here. colors
> 
> checking on it now...
> 
> Edit: May I clarify @chiahou , what you mean by the sequence? Is it that you expect the orange colors (left to right) to be ordered from S21, S64, ..., until S100 (in increasing numbers)? Or is it by increasing shades or based on the Figma coloring [here](https://www.figma.com/file/87fhOdF8vk1Uiy20bCqS85/Sirius-Web---UI-Kit-3.0-%5BQ4-2022%5D?type=design&node-id=4633-49902&t=tbDAnhaVyB5lqNCe-0)? Thank you.
> 
> I believe that the current ordering is based on alphabetical order (in this case, S100 comes first compared to S21, unless S21 is changed to S100), and is the same as the one deployed (the only difference is only in the color that is changed). But let me try to see if we can change the ordering sequence.
> 
> Edit 2: Let me know @chiahou what sequence you prefer there, is it based on the Figma or any other sequence. I can edit those colors ordering, for example (red and orange based on increasing number): image

@michac789 yes, sorting based on increasing number is good enough. Thank you!

Reply: sure!

**Comment by michac789** [comment]
@chiahou 
I hope this is good enough:
image

**Review by chiahou — APPROVED** [review]
all good, thanks

**Comment by michac789** [comment]
replace to new pr: https://github.com/glints-dev/glints-aries/pull/827

**Pull request closed without merging**